<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThresholdSegmentationLSFilterInterface</class>
 <widget class="QWidget" name="ThresholdSegmentationLSFilterInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>213</width>
    <height>568</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>213</width>
    <height>568</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>568</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>650</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Threshold Segmentation Level Set Filter</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="activeHLayout" stretch="1,2">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QLabel" name="activeLabel">
       <property name="text">
        <string>Target:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="activeComboBox">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="maskHLayout" stretch="1,2">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QLabel" name="maskLabel">
       <property name="text">
        <string>Mask:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="maskComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="iterationsHLayout_top">
     <item>
      <widget class="QLabel" name="iterationsLabel">
       <property name="text">
        <string>Iterations</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="iterationsHSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="iterationsHLayout_bottom"/>
   </item>
   <item>
    <widget class="QGroupBox" name="thresholdGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>173</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>173</height>
      </size>
     </property>
     <property name="title">
      <string>Threshold</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="upperHLayout_top">
        <item>
         <widget class="QLabel" name="upperLabel">
          <property name="text">
           <string>Upper</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="upperHSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="upperHLayout_bottom"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="lowerHLayout_top">
        <item>
         <widget class="QLabel" name="lowerLabel">
          <property name="text">
           <string>Lower</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="lowerHSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="lowerHLayout_bottom"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="weightsGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>241</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>241</height>
      </size>
     </property>
     <property name="title">
      <string>Weights</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="curvatureHLayout_top">
        <item>
         <widget class="QLabel" name="curvatureLabel">
          <property name="text">
           <string>Curvature</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="curvatureHSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="curvatureHLayout_bottom"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="propagationHLayout_top">
        <item>
         <widget class="QLabel" name="propagationLabel">
          <property name="text">
           <string>Propagation</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="propagationHSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="propagationHLayout_bottom"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="edgeHLayout_top">
        <item>
         <widget class="QLabel" name="edgeLabel">
          <property name="text">
           <string>Edge</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="edgeHSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="edgeHLayout_bottom"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="replaceRunLayout">
     <item>
      <widget class="QCheckBox" name="replaceCheckBox">
       <property name="text">
        <string>Replace</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="runButton">
       <property name="text">
        <string>Run Filter</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
